cmake/propolis/common.cmake

```diff
@@ -32,11 +32,11 @@ function(propolis_fetch_populate)
         add_subdirectory(${${PROPOLIS_L}_SOURCE_DIR}/src ${${PROPOLIS_L}_BINARY_DIR})
 
         # Removing warnings from freertos compilation on executable target
-        if (DISABLE_EXTERNAL_WARNINGS) 
+        if (DISABLE_EXTERNAL_WARNINGS)
             target_compile_options(swarmus-propolis-pheromones PRIVATE -w)
-            set_target_properties(swarmus-propolis-pheromones PROPERTIES CXX_CLANG_TIDY "" )
-        endif()
+            set_target_properties(swarmus-propolis-pheromones PROPERTIES CXX_CLANG_TIDY "")
+        endif ()
 
-    endif()
+    endif ()
```

Comment:
Why the weird spacing after ifs?

Reply:
I blame @dubh3401.
Gonna revert all changed cmakes.